### PR TITLE
New Console tutorial and docs updates

### PR DESCRIPTION
**Pull request by SariSaar**

### CODE REVIEW

File: src/docs/tutorial-branding/first-edit/index.md [hunk 1]
```diff
@@ -1,63 +1,167 @@
...
+The &:nth-of-type(odd) selector targets elements that are the
+odd-numbered siblings within their parent container. Therefore, the
+background-colour property is applied to every second element on the
+landing page.
+
+Let’s change the background colour to a new colour. First, define a new
+variable:
+
+```css
+var(—colorPastelPurple)
```
Comment (SariSaar) [inline src/docs/tutorial-branding/first-edit/index.md:127]: I'm confused by the instructions here 😄
- where do I define the variable
- why is the variable `-colorPastelPurple` but the var used in the class below is `--colorPurple`
@shareoc would you be able to clarify this?

**Comment by SariSaar** [comment]
The history looks choppy because I failed with a rebase , and apparently there's a bunch of changed files with only formatting changes, but let's do a squash merge to keep the master branch from getting messed up

File: src/docs/tutorial-branding/first-edit/index.md [hunk 2]
```diff
@@ -1,63 +1,167 @@
...
-You should notice that some text and background-colors were changed to
-green.
+Don’t confuse the class attribute of HTML elements and the className
+prop of React components!
 
-![Hero section after changing the marketplace color](./marketplace-color-changed.png)
+</info>
 
-**Congratulations!** You have made your first customization.
+## Summary
```
Comment (SariSaar) [inline src/docs/tutorial-branding/first-edit/index.md:164]: Summary has no content yet

File: src/docs/tutorial-branding/working-with-microcopy/index.md [hunk 3]
```diff
@@ -106,6 +123,8 @@ on the search page:
 
 ![Topbar with updated microcopy](./updated_topbar_microcopy.png)
 
+## Further reading
+
```
Comment (SariSaar) [inline src/docs/tutorial-branding/working-with-microcopy/index.md:127]: Do we want to add a summary here?

File: src/docs/tutorial-branding/configuration/index.md [hunk 4]
```diff
@@ -130,6 +115,21 @@ other variables:
...
+Another configuration defined in configDefault.js is **siteTitle** - and
+then that variable is used in `<meta>` tags and
+[webpage's schema](https://schema.org/).
+
+```js
+// Site title is needed in meta tags (bots and social media sharing reads those)
+const siteTitle = 'Biketribe';
+```
+
+### Task: _Set listing's minimum price_
```
Comment (SariSaar) [inline src/docs/tutorial-branding/configuration/index.md:127]: This is set in Console, and if the value exists in both Console/asset and locally, the asset value gets used. Should we omit the listing minimum price setting, or describe how to set it in Console?

File: src/docs/tutorial-branding/first-edit/index.md (hunk 1 above)
Comment (shareoc) [reply src/docs/tutorial-branding/first-edit/index.md:127]: I'm currently going through the article and changing the colours to match the new Sauna time theme so it's WIP! @SariSaar

File: src/docs/tutorial-branding/configuration/index.md (hunk 4 above)
Comment (shareoc) [reply src/docs/tutorial-branding/configuration/index.md:127]: I think we just replace that with a picture of Console accompanied by a text that you can change the listing price in Console. I can handle that. @SariSaar

File: src/docs/tutorial-branding/working-with-microcopy/index.md (hunk 3 above)
Comment (shareoc) [reply src/docs/tutorial-branding/working-with-microcopy/index.md:127]: 👍 I can handle that

File: src/docs/ftw-configuration/hosted-and-local-configurations/index.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+logo, listing types, listing fields, and minimum transaction size. Those
+configurations are read from the hosted assets in the template by
+default. This means that if you want to, for instance, add a listing
+type with a custom transaction process, or add a custom listing field in
+the template code, you will need to modify how configurations get merged
+in the template.
+
+## How configurations are merged in the template
+
+The template uses a function [mergeConfig]() in
```
Comment (SariSaar) [inline src/docs/ftw-configuration/hosted-and-local-configurations/index.md:28]: TODO Add link to repo once template changes released!

File: src/docs/concepts-development/console-overview/index.md [hunk 6]
```diff
@@ -96,11 +96,12 @@ your marketplace ID, URL and name. Here, you can also configure an
...
-add your existing microcopy file content from your Sharetribe Web
-Template, or copy and paste the default language-specific microcopy file
-from the featured links, and modify the texts to suit your marketplace.
-Read more about [modifying microcopy in Console](/concepts/microcopy/).
+**Content** is where you can modify your marketplace content and
+configurations. You can
+[modify your marketplace microcopy](/concepts/microcopy/), create
+[content pages](/concepts/headless-content-management/), and configure
+aspects of your marketplace branding, layout, and other details
+[without coding]().
```
[the remark this answers is not in the document]
Comment (shareoc) [reply src/docs/concepts-development/console-overview/index.md:104]: Maybe this should just link to first part of the tutorial – I can't think of a suitable article in the CA docs: https://drive.google.com/drive/u/0/folders/1lgv2pOXQlC9D6tTtXTyn_J5H5J9VEoNY

@SariSaar

[the remark this answers is not in the document]
Comment (SariSaar) [reply src/docs/concepts-development/console-overview/index.md]: Good idea! Will update 👍

**Pull request merged**